packages/next-swc/crates/napi/src/lib.rs

```diff
@@ -73,19 +76,84 @@ shadow_rs::shadow!(build);
 static ALLOC: turbopack_binding::turbo::malloc::TurboMalloc =
     turbopack_binding::turbo::malloc::TurboMalloc;
 
+static LOG_FILE_MUTEX: Mutex<()> = Mutex::new(());
+static LOG_THROTTLE: Mutex<SystemTime> = Mutex::new(SystemTime::UNIX_EPOCH);
```

Comment:
Nit: Instant is typically preferred for measuring duration, though I get that this would need to be an `Option<Instant>` and it might make the code slightly more complicated.

This code later might panic:

```
last_error_time.elapsed().unwrap().as_secs()
```

because `Duration` cannot be negative, and `SystemTime` is not monotonic.